src/dropdown/dropdown.service.ts

```diff
@@ -56,10 +59,9 @@ export class DropdownService {
 
 		this.menuInstance = dropdownWrapper;
 
-		this.positionDropdown(parentRef, dropdownWrapper);
-		this.resize = fromEvent(window, "resize")
-			.pipe(throttleTime(100))
-			.subscribe(() => this.positionDropdown(parentRef, dropdownWrapper));
+		this.animationFrameSubscription = this.animationFrameService.tick.subscribe(() => {
+			this.positionDropdown(parentRef, dropdownWrapper);
+		});
```

Comment:
What about?
```js
this.animationFrameSubscription = this.animationFrameService.tick
   .subscribe(() => this.positionDropdown(parentRef, dropdownWrapper));
```

Reply:
More or less the same thing ... Currently it's marginally more readable to my eyes